build/Helix/RunTestsInHelix.proj

```diff
@@ -11,271 +11,281 @@
   <ItemGroup>
     <HelixCorrelationPayload Include="..\..\HelixPayload\$(Configuration)\$(Platform)" />
     
-    <HelixWorkItem Include="InteractionTests.ColorPickerTests">
+    <HelixWorkItem Include="InteractionTests.ColorPickerTests" Condition=" '$(JobName)'=='RunTestsInHelix' ">
```

Comment:
I don't like take a dependency on the job name. The yml file is already passing in a set of parameters to the proj (eg. the set of queues to run on), so just update that to send an extra parameter (e.g. "RunNugetTestSuite").

Reply:
Added a `TestSuite` parameter.